Libs/MRML/Core/vtkMRMLTransformNode.h

```diff
@@ -120,29 +120,35 @@ class VTK_MRML_EXPORT vtkMRMLTransformNode : public vtkMRMLDisplayableNode
 
   ///
   /// Get concatenated transforms to world.
+  /// Resulting transformToWorld multiply order will be overwritten to PostMultiply
   /// \sa GetTransformBetweenNodes
   void GetTransformToWorld(vtkGeneralTransform* transformToWorld);
 
   ///
   /// Get concatenated transforms from world.
+  /// Resulting transformFromWorld multiply order will be overwritten to PostMultiply
   /// \sa GetTransformBetweenNodes
   void GetTransformFromWorld(vtkGeneralTransform* transformToWorld);
+  void GetTransformFromWorld(vtkGeneralTransform* transformFromWorld);
```

Comment:
Is this change intentional?

Reply:
yes

Reply:
This line causes a build error. You need to remove the old line.

Reply:
Oh, I redid the commit so many times that that slipped through